internal/servers/controller/controller.go

```diff
@@ -124,6 +127,32 @@ func New(ctx context.Context, conf *Config) (*Controller, error) {
 		}
 	}
 
+	clusterListeners := make([]*base.ServerListener, 0)
```

Comment:
I'm assuming that you're pulling the loop iteration out of elsewhere in the code. Can you do it in the same PR? If it happens across two PRs it's much harder to assess whether the code that is migrating here will have the same effect as the code removed elsewhere.

Reply:
Yep, just merged these changes onto another [PR](https://github.com/hashicorp/boundary/pull/1907) where they're being used.